src/style.css

```diff
@@ -1,3 +1,86 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --active-collor: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav-bar {
+  display: flex;
+  margin: 0 50px 0 0;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+ul :first-child a {
+  margin: 0 10px 0 0;
```

Comment:
If you want to apply a margin on one side, you can use the `margin-left` or `margin-right` property.